pom.xml

```diff
@@ -145,35 +145,35 @@
     </distributionManagement>
 
     <properties>
-        <dynamic.build.properties.version>0.0.1.a2</dynamic.build.properties.version>
+        <dynamic-build-properties-maven-plugin.version>0.0.1.a2</dynamic-build-properties-maven-plugin.version>
 
         <javase.version>${jdk.version}</javase.version>
         <!-- BOM-referenced versions -->
 
         <osgi-resource-locator.version>1.0.3</osgi-resource-locator.version>
         <snakeyaml.version>2.2</snakeyaml.version>
-        <jstl-impl.version>3.0.1</jstl-impl.version>
-        <mojarra.version>4.0.0.payara-p2</mojarra.version>
+        <jakarta.servlet.jsp.jstl.version>3.0.1</jakarta.servlet.jsp.jstl.version>
+        <jakarta.faces.version>4.0.0.payara-p2</jakarta.faces.version>
```

Comment:
I'm not sure about this change. I think our convention should be that we use project names where they exist.
So Mojarra should stay as `mojarra.version`, EclipseLink should stay as `eclipselink.version`.

We can then extend this for any sub-project artefacts e.g. eclipselink.asm.version

This helps to differentiate the properties from their specification API versions e.g. Jakarta Faces API version does not necessarily align with the Jakarta Faces implementation (Mojarra) version.